packages/util/src/bytes.ts

```diff
@@ -1,8 +1,33 @@
 import BN from 'bn.js'
-import { intToBuffer, stripHexPrefix, padToEven, isHexString, isHexPrefixed } from 'ethjs-util'
+import { stripHexPrefix, padToEven, isHexString, isHexPrefixed } from 'ethjs-util'
 import { PrefixedHexString, TransformableToArray, TransformableToBuffer } from './types'
 import { assertIsBuffer, assertIsArray, assertIsHexString } from './helpers'
 
+/**
+ * Converts a `Number` into a hex `String`
+ * @param {Number} i
+ * @return {String}
+ */
+export const intToHex = function (i: number) {
+  if (typeof i !== 'number' || isNaN(i) || i === Infinity || i < 0 || i % 1 !== 0) {
```

Comment:
should we add `|| !Number.isSafeInteger(i)`?

Reply:
Yep good point!

Reply:
It seems like I can directly use this method instead of checking all these conditions :smile:

Reply:
Cool, didn't know about that method!

I tested the conditions and they account for all of them except `i < 0`

Reply:
Yup, just pushed the changes that accounts for it :)